xml/System/DateOnly.xml

```diff
@@ -1364,11 +1387,12 @@
         <Parameter Name="result" Type="System.DateOnly" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="formats">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">The span containing the string to parse.</param>
+        <param name="formats">An array of allowable formats of s.</param>
+        <param name="result">When this method returns, contains the DateOnly value equivalent to the date contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s parameter is Empty, or does not contain a valid string representation of a date. This parameter is passed uninitialized.</param>
+        <summary>Converts the specified char span of a date to its DateOnly equivalent and returns a value that indicates whether the conversion succeeded.</summary>
+        <returns>
+          <see langword="true" /> if the s parameter was converted successfully; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if<paramref name="s" /> was converted successfully; otherwise, <see langword="false" />.</returns>
```